src/main/java/blusunrize/immersiveengineering/common/blocks/metal/CrusherBlockEntity.java

```diff
@@ -342,6 +357,10 @@ public boolean isInWorldProcessingMachine()
 	@Override
 	public boolean additionalCanProcessCheck(MultiblockProcess<CrusherRecipe> process)
 	{
+		if (processQueue.size() > 0 && processQueue.get(0).getRecipe(getLevelNonnull()) != null)
+		{
+			spawnParticles(processQueue.get(0).getRecipe(getLevelNonnull()).input.getItems()[0]);
+		}
```

Comment:
This doesn't seem like the right place for spawning particles. I'd suggest overriding `tickServer` and spawning the particles from there.

Reply:
Should be fixed in newest commit